core/src/components/picker-legacy-column/picker-column.tsx

```diff
@@ -6,13 +6,13 @@ import { getClassMap } from '@utils/theme';
 
```

Comment:
> The ticket & design doc were unclear about whether or not this file & others like it should be renamed to e.g. picker-legacy-column.tsx. I don't see any reason it would matter, I just wanted to check that we aren't renaming them.

This is addressed at the top of the PR:

"I intentionally did not change the file names for the picker and picker column. There should be no naming collisions with the internal picker since these are in separate directories. Also, the names of the files are internal details and do not impact how developers build Ionic apps."

> As a side question: I assume we do want to rename e.g. picker-internal-column.tsx to picker-column.tsx?

Yea I think that makes sense.